lwmem/src/lwmem/lwmem.c

```diff
@@ -677,14 +687,15 @@ lwmem_region_t regions[] = {
  *                      It must be called only once to setup memory regions
  */
 size_t
-lwmem_assignmem_ex(lwmem_t* const lw, const lwmem_region_t* regions) {
+lwmem_assignmem_ex(lwmem_t* lw, const lwmem_region_t* regions) {
+    lw = LWMEM_GET_LW(lw);
```

Comment:
Initialization variables first, then assignment operator. Please move this after